sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemRequestOptions.java

```diff
@@ -39,6 +39,7 @@ public class CosmosItemRequestOptions {
     private CosmosDiagnosticsThresholds thresholds;
     private Boolean nonIdempotentWriteRetriesEnabled;
     private boolean useTrackingIds;
+    private CosmosEndToEndOperationLatencyPolicyConfig endToEndOperationLatencyPolicyConfig ;
```

Comment:
You'll also have to make the changes to allow end2end operation timeout for CosmosBatchRequestOptions

Reply:
@FabianMeiswinkel  - wondering if we can keep batch / bulk / patch out of this week's release. Let them come in a follow up PR? To speed up process.

Reply:
Created an issue. https://github.com/Azure/azure-sdk-for-java/issues/34722
@kushagraThapar  assigned to you for now. Please add any more tags if required and I can take it up later